ods_ci/tests/Tests/650__distributed_workloads/test-smoke.robot

```diff
@@ -72,6 +70,9 @@ Training operator smoke test
     [Documentation]    Check that Training operator deployment is up and running
     [Tags]    Smoke
     ...       DistributedWorkloads
+
```

Comment:
Maybe it would be better to use Setup and Teardown keywords as are mentioned here. It will make sure that tear down is run even if test fails. https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#test-setup-and-teardown

Reply:
Added the suggestion